force_bdss/core_driver_events.py

```diff
@@ -46,6 +46,19 @@ def serialize(self):
         return self.parameter_names + self.kpi_names
 
 
+class WeightedMCOStartEvent(MCOStartEvent):
+    """Initializes reporting of weights generated during an MCO by a
+    WeightedOptimizerEngine"""
+
+    def serialize(self):
```

Comment:
We don't use the `serialize` method of the parent class here, do we? Would you say this is because we want a certain format of the header, in particular `parameters` + `(kpi, kpi_weight)`?

Reply:
Yes, I believe in the original table display each weight was reported in a column directly to the right of the KPI, rather than being appended to the end.

I just overloaded the function rather than calling the super to put this for both `Weighted*` events